### PR TITLE
finished coding explorer section

### PR DESCRIPTION
**Pull request by AraKakai**
Done coding the explorer section.
Screenshot 2023-08-28 102658

### CODE REVIEW

**Comment by melmarieee** [comment]
please make your description imperative something like "add coding section location about page" and please update the pr description for your output description and screenshot, thanks.

File: css/explorer.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+body {
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    min-height: 100vh;
```
Comment (melmarieee) [inline css/explorer.css]: put it in the container class instead.

File: css/explorer.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  }
+
+  .container {
+    display: flex;
+    max-width: 800px;
+    background-color: white;
+    border-radius: 8px;
+    overflow: hidden;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  }
```
Comment (melmarieee) [inline css/explorer.css]: how about we use a bootstarp container?

File: index.html [hunk 3]
```diff
@@ -4,9 +4,18 @@
...
 <body>
-    
+    <div class="container">
+    <div class="column left-column">
+      <h1> Did you know? </h1>
+      <p>It is often referred to as the "Last Lung of Manila" due to its significance in providing much-needed greenery and biodiversity within the urban landscape. The park is known for its diverse collection of trees, including around 3,000 trees from over 60 different species, making it an important haven for various plant and bird species in the midst of the city's concrete jungle. </p>
+      -Philippine Daily Inquirer (2017)</p> 
+    <a href="#"> Read More > </a> </div>
+    <div class="column right-column">
+      <img src="media/last lung.jpg" alt="Image">
```
Comment (melmarieee) [inline index.html]: please use the img-fluid class in bootstrap for this to be responsive.

File: index.html [hunk 4]
```diff
@@ -4,9 +4,18 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
```
Comment (melmarieee) [inline index.html:6]: please add a bootstarp links in here.

File: index.html [hunk 5]
```diff
@@ -4,9 +4,21 @@
...
-    <link rel="stylesheet" href="/css/index.css">
-</head>
+    <link rel="stylesheet" href="	https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/css/bootstrap.min.css">
+    <link rel="stylesheet" href="css/explorer.css">
+  </head>
```
Comment (melmarieee) [inline index.html:9]: [nit] the space

File: index.html [hunk 6]
```diff
@@ -4,9 +4,21 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
-    <link rel="stylesheet" href="/css/index.css">
-</head>
+    <link rel="stylesheet" href="	https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/css/bootstrap.min.css">
```
Comment (melmarieee) [inline index.html]: [nit] please remove the unecessary space.

File: index.html [hunk 7]
```diff
@@ -4,9 +4,21 @@
...
 <body>
-    
+    <section class="explorer">
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-9"> 
+            <h1> Did you know? </h1>
+            <p>It is often referred to as the "Last Lung of Manila" due to its significance in providing much-needed greenery and biodiversity within the urban landscape. The park is known for its diverse collection of trees, including around 3,000 trees from over 60 different species, making it an important haven for various plant and bird species in the midst of the city's concrete jungle. </p>
+            -Philippine Daily Inquirer (2017)</p> 
+          <a href="#"> Read More > </a> </div>
```
Comment (melmarieee) [inline index.html]:
```diff
- <div class="col-lg-9"> 
-            <h1> Did you know? </h1>
-            <p>It is often referred to as the "Last Lung of Manila" due to its significance in providing much-needed greenery and biodiversity within the urban landscape. The park is known for its diverse collection of trees, including around 3,000 trees from over 60 different species, making it an important haven for various plant and bird species in the midst of the city's concrete jungle. </p>
-            -Philippine Daily Inquirer (2017)</p> 
-          <a href="#"> Read More > </a> </div>

+<div class="col-lg-9"> 
+  <h1> Did you know? </h1>
+  <p>It is often referred to as the "Last Lung of Manila" due to its significance in providing much-needed greenery and biodiversity within the urban landscape. The park is known for its diverse collection of trees, including around 3,000 trees from over 60 different species, making it an important haven for various plant and bird species in the midst of the city's concrete jungle. </p> -Philippine Daily Inquirer (2017)</p> 
+  <a href="#">Read More></a>
+</div>
```

File: index.html [hunk 8]
```diff
@@ -4,9 +4,21 @@
...
-</head>
+    <link rel="stylesheet" href="	https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/css/bootstrap.min.css">
+    <link rel="stylesheet" href="css/explorer.css">
+  </head>
 <body>
-    
+    <section class="explorer">
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-9">
```
Comment (melmarieee) [inline index.html]: lets use `md` instead of `lg` for medium size devices, most of us use medium sizes? WDYT? 🤔

**Comment by AraKakai** [comment]
Fixed the current explorer section as per Ms. Mel.
-xoxo
Screenshot 2023-08-28 211911

**Review by melmarieee — APPROVED** [review]
LGTM :)

**Pull request merged**